tutorials/simple_time_series_example.ipynb

```diff
@@ -0,0 +1,406 @@
+    "\n",
+    "- a pandapipes network\n",
+    "- predefined time series for the sinks and sources in the network (e.g. 2 csv files)\n",
+    "\n",
+    "First, the following imports must be carried out for a general time series simulation:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import pandas as pd\n",
+    "import pandapower.control as control\n",
+    "import pandapipes\n",
+    "from pandapower.timeseries import DFData\n",
+    "from pandapower.timeseries import OutputWriter\n",
+    "from pandapipes.timeseries import run_timeseries_ppipe\n",
+    "from pandapipes import pp_dir"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since in this example an already existing network is loaded, the upcoming code must be added:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pandapipes import networks"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now the pandapipes network can be created/ loaded:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "net = pandapipes.networks.simple_gas_networks.gas_meshed_delta()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is the delta STANET network from the chapter [Meshed Networks](https://pandapipes.readthedocs.io/en/latest/networks/meshed/meshed_networks.html) with the medium gas."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src=\"pics/time_series_simple_example_network.png\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Furthermore, the network can be prepared. For this purpose the time series for sinks and sources have to be defined. Since there is only one sink and one source, the respective csv file only consists of one column. The number of lines corresponds to the desired number of time steps, in this case 10."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src=\"pics/time_series_simple_example_csv_files.png\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With the code"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "profiles_sink = pd.read_csv(os.path.join(pp_dir, 'test', 'pipeflow_internals', 'data',\n",
+    "                                         'test_time_series_sink_profiles.csv'), index_col=0)\n",
+    "profiles_source = pd.read_csv(os.path.join(pp_dir, 'test', 'pipeflow_internals', 'data',\n",
+    "                                           'test_time_series_source_profiles.csv'), index_col=0)\n",
```

Comment:
Are these the right files? They differ from the picture, have more columns (6) and the numbers are much smaller. Please check again